### PR TITLE
F whats new stable

### LINKED ISSUE #1132: GCode auto deletion notification not showing on stable [/2]
@gallore

### PR DESCRIPTION
**Pull request by khaledsherkawi**
- Adding the code deletion to the stable whats new wizard
- Modifying the beta whats new wizard for cleaner code

### CODE REVIEW

**Comment by andywerner** [comment]
One question that I haven't thought before:
What If a user already enabled GCode auto deletion in v0.8.0?

My thoughts:
- they do not get this part of the What's New screen
- the checkbox in the GCode deletion part of the What's New screen reflects the current setting (is check, if auto deletion is already enabled)
Both solutions seem fine to me. Any other ideas?

File: octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2 [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
-    <p>{{ _('You will feel an improvement specially when using dark materials or with difficult light conditions, thanks to the self-adjusting shutter speed. ') }}</p>
-    <p>{{ _('Check the status of your camera in the new camera settings panel:') }}</p>
-    <img class="wizard-big-img wizard-shadowed-img" src="/plugin/mrbeam/static/img/whatsnew_dialog/camera_status.png">
+    <form class="form-horizontal">
+        <div class="control-group">
+            <div>
+                <p>{{ _('Every time you run a laser job, a GCode file is automatically created. We recommend to enable the auto-deletion of GCode files so that your Mr Beam doesn\'t get too loaded and it can work faster. With this enabled, only the last 25 GCode files will be stored.') }} </p>
+                <p>{{ _('You can alternativelly delete the GCode files manually in the Design Library.') }} </p>
+                <label class="checkbox">
+                    <input type="checkbox" data-bind="checked: gcodeAutoDeletion">
```
Comment (andywerner) [inline octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2]: I'm surprised that this works. @khaledsherkawi knowing you, I'm sure you tested it. Still, can you please confirm, that it works?

**Review by andywerner — APPROVED** [review]
Please check my comment

File: octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2 (hunk 1 above)
Comment (khaledsherkawi) [reply octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2]: It's the same concept you used for the beta whats new where I changed the IDs of both elements (beta whats new and stable whats new) and bind them to the WizardGcodeDeletionViewModel.

and yes of course it has been tested over and over.

**Pull request merged**